tools-local/unix/cmake/get-version.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Gets the version of CMake at the specified path.
+
+usage()
+{
+    echo "Usage: $0 cmake ToolName --ToolPath <path>"
```

Comment:
A thought I had after our talk offline about optional params: I think we could get rid of the names. We could instead pass empty string to indicate that the option isn't wanted. Ex:

```sh
|> p() { echo $1; echo $2; echo $3; }
|> set -x
|> foo=
|> p a "$foo" c
+ p a '' c
+ echo a
a
+ echo

+ echo c
c
```

We'd only need a single place for named args, then: `build-native.sh`, where there's already an arg parse loop.

(For testing scripts directly, typing `''` in a shell is an empty string and it's treated this way.)

@ravimeda @MichaelSimons Thoughts?